### PR TITLE
D3ASIM-3596: Fix slot_completion sent to myco matcher

### PR DESCRIPTION
**Pull request by Hassan754**

### CODE REVIEW

File: src/d3a/d3a_core/simulation.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
                 bid_offer_matcher.event_tick(
                     is_it_time_for_external_tick=global_objects.external_global_stats.
                     is_it_time_for_external_tick(current_tick_in_slot),
-                    slot_completion=f"{int(self.progress_info.percentage_completed)}%",
+                    slot_completion=f"{int((tick_no/config.ticks_per_slot) * 100)}%",
```
Comment (hannesdiedrich) [inline src/d3a/d3a_core/simulation.py:380]: Can you please explain, why the `self.progress_info.percentage_completed` can not be used any more? Are the values wrong?

Comment (Hassan754) [reply src/d3a/d3a_core/simulation.py:380]: This is the percentage completed compared to the whole simulation, what Chris asked for is the percentage completed in a market cycle

**Review by chrisd12 — APPROVED** [review]
LGTM

File: src/d3a/d3a_core/simulation.py (hunk 1 above)
Comment (hannesdiedrich) [reply src/d3a/d3a_core/simulation.py:380]: I see, thanks a lot!

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**